### PR TITLE
Improve website style on Arabic

### LINKED ISSUE #251: [BUG] Paragraphs and headlines not aligned correctly in Arabic version
### Description of the bug

**Description of the bug**
Arabic paragraphs and section headlines are misaligned in the Arabic version.

**How to reproduce**
Change the language in the translation button to Arabic

**Expected behavior**
Arabic paragraphs and section headlines text should be aligned right.

**Screenshots**
Paragraph-ar-alignment-02
Paragraph-ar-alignment

### Add screenshots of the bug (if applicable)

Paragraph-ar-alignment

### Additional information/context

_No response_

### PR DESCRIPTION
**Pull request by ahmed0saber**
Closes #251 

**What change does this pull request introduce?**
Improving the style of TheAlgorithms website on Arabic, by adjusting the direction to "rtl", replacing left and right of margin and padding to inline-start and inline-end, forcing "ltr" direction in some places, and finally adding some custom styling if required.

**Screenshots**
screencapture-3000-thealgorithms-website-q2fhlgkhxd5-ws-eu110-gitpod-io-ar-2024-04-28-21_47_43

**Checklist**

- [x] I worked on a branch other than `main`.
- [x] My branch is up-to-date with the Upstream `main` branch.
- [x] I have fixed potential errors using `yarn lint`.
- [x] I ran `yarn build` to check everything still builds successfully.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -129,3 +129,24 @@ html::-webkit-scrollbar {
...
+}
+
+html[lang=ar] .Jumbo_SearchBar__form label[data-shrink=true]{
+  transform: translate(-12px, 10px) scale(0.75);
+}
+
+html[lang=ar] .AlgorithmCard__div nav,
+html[lang=ar] .AlgorithmCard__div h2 {
+  direction: ltr;
+}
```
Comment (maltejur) [inline styles/globals.css]: Could you move the styles you added here to the respective `style.module.css` files instead?

**Review by maltejur — CHANGES_REQUESTED** [review]
Thanks, this seems like a great change. One small thing.

File: styles/globals.css (hunk 1 above)
Comment (ahmed0saber) [reply styles/globals.css]: I have moved them.
Learnt something new today :)

**Comment by ahmed0saber** [comment]
Hello @maltejur , hope you are doing well.
I have moved the global styles of Arabic and RTL into respective CSS modules, and fixed a small issue I have found with the translate icon in side menu of navbar.

**Review by maltejur — APPROVED** [review]
Looks great now, thanks!

**Pull request merged**